python/tank/platform/qt/event_emitter.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from . import QtCore
+
+class EventEmitter(QtCore.QObject):
+    """
+    A container object for event signals.
+
+    :signal event(object): A generic event signal housing a Python object.
+        Typically this object will be an :class:`sgtk.platform.EngineEvent`
+        object.
+    """
+    event = QtCore.Signal(object)
```

Comment:
Thinking about stuff like maya's event system - which we want to hook into in order to send out events (or events system is just a generic broker for in-DCC event systems) - thinking that there may be clashes between maya, QT and our stuff - maybe this is just fine and i am being a bit paranoid, but I insinctively feel that if there aren't any obvious benefits with using qt signals/slots for this, we should explore what a solution without QT would look like.